### PR TITLE
Add signUp component and styles.

### PR DESCRIPTION
**Pull request by olhavalchak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,5 +42,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^8.5.0",
+    "eslint-plugin-react": "^7.27.1"
```
Comment (AndreiYar) [inline package.json]: "devDependencies": {
    "eslint": "^7.32.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-plugin-import": "^2.25.1",
    "eslint-plugin-jsx-a11y": "^6.4.1",
    "eslint-plugin-react": "^7.26.1",
    "eslint-plugin-react-hooks": "^4.2.0"
  }

File: package.json [hunk 2]
```diff
@@ -7,12 +7,14 @@
...
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.24.0",
+    "eslint-config-airbnb": "^19.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
     "redux": "^4.1.2",
+    "save-dev": "0.0.1-security",
```
Comment (AndreiYar) [inline package.json]: Remove

File: package.json [hunk 3]
```diff
@@ -7,12 +7,14 @@
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.24.0",
+    "eslint-config-airbnb": "^19.0.2",
```
Comment (AndreiYar) [inline package.json]: Move to the dev deps

File: .eslintrc.json [hunk 4]
```diff
@@ -23,6 +24,12 @@
     "rules": {
         "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
         "import/prefer-default-export": "off",
-        "linebreak-style": 0
+        "linebreak-style": 0,
+        "react/function-component-definition": [
```
Comment (AndreiYar) [inline .eslintrc.json]: Remove

File: src/theme/styles/index.css [hunk 5]
```diff
@@ -0,0 +1,393 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (AndreiYar) [inline src/theme/styles/index.css]: Rename to normalize.scss
Create index.scss -> @import './normalize.scss'

File: src/pages/SignUp.jsx [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { SignUpForm } from '../components/forms';
+
+export function SignUpPage() {
```
Comment (AndreiYar) [inline src/pages/SignUp.jsx]: Use arrow function

File: src/components/forms/SignUpForm/index.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export function SignUpForm() {
+  return (
+    <div className="loginBox">
+      <h1>Sign Up</h1>
+
+      <form className="form">
+        <input
+          className="input"
```
Comment (AndreiYar) [inline src/components/forms/SignUpForm/index.jsx:12]: Use please styled component.

File: package.json (hunk 1 above)
Comment (olhavalchak) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (olhavalchak) [reply package.json]: done

File: package.json (hunk 3 above)
Comment (olhavalchak) [reply package.json]: done

**Pull request closed without merging**